yarn-project/simulator/src/client/simulator.test.ts

```diff
@@ -42,16 +42,16 @@ describe('Simulator', () => {
     const contractAddress = AztecAddress.random();
     const nonce = Fr.random();
     const storageSlot = Fr.random();
-    const noteTypeId = Fr.random();
+    const noteTypeId = new Fr(8411110710111078111116101n); // TokenNote
```

Comment:
Should we have a utility of something for computing this one? 🤔

Reply:
Ah good idea, I'll do this in a separate PR.